numeric.c

```diff
@@ -3366,9 +3408,30 @@ int_anybits_p(VALUE num, VALUE mask)
 
 /*
  *  call-seq:
- *     int.nobits?(mask)  ->  true or false
+ *    nobits?(mask) -> true or false
+ *
+ *  Returns +true+ if no bit that is set (=1) in +mask+
+ *  is also set in +self+; returns +false+ otherwise.
+ *
+ *  Example values:
+ *
+ *    0b00000000  self
```

Comment:
I would cut this example and leave the other two.

Reply:
Done.